tests/src/test/java/alluxio/server/ft/journal/TableMasterJournalIntegrationTest.java

```diff
@@ -49,9 +51,12 @@
  * Integration tests for table master functionality.
  */
 public class TableMasterJournalIntegrationTest {
+  private static final int WORKER_CAPACITY_BYTES = 200 * Constants.MB;
   @ClassRule
   public static LocalAlluxioClusterResource sClusterResource =
-      new LocalAlluxioClusterResource.Builder().setNumWorkers(1).build();
+      new LocalAlluxioClusterResource.Builder()
+          .setProperty(PropertyKey.WORKER_MEMORY_SIZE, WORKER_CAPACITY_BYTES)
```

Comment:
Why does this need to be set? The default in `ConfigurationTestUtils.java` is `100m`.